src/server/acl/helpers.cc

```diff
@@ -318,4 +318,26 @@ ParseAclSetUser<std::vector<std::string_view>&>(std::vector<std::string_view>&,
 
 template std::variant<User::UpdateRequest, ErrorReply> ParseAclSetUser<CmdArgList>(
     CmdArgList args, const CommandRegistry& registry, bool hashed, bool has_all_keys);
+
+std::string AclKeysToString(const AclKeys& keys) {
+  if (keys.all_keys) {
+    return "~*";
+  }
+  std::string result;
+  for (auto& glob : keys.key_globs) {
+    auto& [pattern, op] = glob;
+    if (op == KeyOp::READ_WRITE) {
+      absl::StrAppend(&result, "~", pattern, " ");
+      continue;
+    }
```

Comment:
So always the compact form is used, %RW never appears in output? So they didn't decide to deprecate just ~?

Reply:
Yes! Honestly, I would deprecate the `RW` instead, because `~` is shorter :stuck_out_tongue: